build.env

```diff
@@ -36,7 +36,7 @@ SNAPSHOT_VERSION=v3.0.1
 HELM_VERSION=v3.1.2
 
 # minikube settings
-MINIKUBE_VERSION=v1.14.1
+MINIKUBE_VERSION=v1.16.0
```

Comment:
why not use `latest`?

Reply:
@obnoxxx currently, we are sticking to the available release once the version is tested in the CI.  if there is any regression in the latest available release we might end up in the CI issues which could block the merging of the PR.